Remove references to "node" in EntityAliasTypeBase

randomletters opened the pull request:
Replace them with "entity".

randomletters committed:
Remove references to "node" in comments and variables.  Replace with "entity".

randomletters commented:
I do not believe the test failures have anything to do with my edit.  The only user facing change was changing the words "node, nodes" to "entity, entities" which does not appear to be the cause of the failure.

Berdir commented on src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php:
```diff
@@ -267,12 +267,12 @@ protected function bulkUpdate(array $ids, array $options = array()) {
 
     if (!empty($options['message'])) {
-      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 node.', 'Updated URL aliases for @count nodes.'));
+      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 entity.', 'Updated URL aliases for @count entities.'));
```
We shouldn't hardcode entity either.. the problem is that there is no generic plural entity label. You could try to rewrite it into something like "for type %label" or so...

randomletters committed:
Change wording of message based on feedback

randomletters commented:
I updated the wording based on your suggestion.

Berdir commented on src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php:
```diff
@@ -267,12 +267,12 @@ protected function bulkUpdate(array $ids, array $options = array()) {
 
     if (!empty($options['message'])) {
-      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 node.', 'Updated URL aliases for @count nodes.'));
+      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated 1 URL alias for type %label.', 'Updated @count URL aliases for type %label.'), array('%label' => $entity->getLabel()));
```
$entity->getLabel() is e.g. a specific node title, $entity is the last entity from the loop above, that's not what you want. Instead, you need to get the entity type definition from the entity manager and then the label of that.

randomletters committed:
Change label to EntityAliasTypeBase label.

randomletters commented:
I changed the label to $this->getLabel() and made the wording of the message less awkward.  This should result in messages similar to the following:

Updated 1 Content URL alias.
Updated 4 User URL aliases.
Updated 2 Taxonomy term paths URL aliases.

The last one is slightly awkward because the label for the taxonomy term alias plugin is inconsistent.  I think it should be changed to "Taxonomy term" or at least kept singular "Taxonomy term path".

Berdir commented:
Works for me, merged.
